fuzz/fuzz_atpointer.cpp

```diff
@@ -0,0 +1,68 @@
+#include "simdjson.h"
+#include "FuzzUtils.h"
+#include <cstddef>
+#include <cstdint>
+#include <string>
+#include <string_view>
+
+struct FuzzData {
+    std::string_view json_pointer;
+    std::string_view json_doc;
+};
+
+/**
+ * @brief split split fuzz data into a pointer and a document
+ * @param Data
+ * @param Size
+ * @return
+ */
+FuzzData split(const char *Data, size_t Size) {
+
+    using namespace std::literals;
+    constexpr auto sep="\n~~~\n"sv;
+
+    std::string_view all(Data,Size);
+    auto pos=all.find(sep);
+    if(pos==std::string_view::npos) {
```

Comment:
Are there `SIMDJSON_PADDING` extra bytes after the end of the document? It looks like we just take whatever's being passed in, and I don't know whether the LLVM fuzzer pads the input ...

Reply:
There is no padding added by libFuzzer, which is a good feature. It means address sanitizer is likely to catch illegal reads. However, I see that I the parse() function mandates padded input! How could this not be detected when running the fuzzer? I will have to investigate this and come back.